From aadb7704628c1b11b2a96437bfafbdf1c385d7de Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:43:16 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(f41cba92)..lineage-16.0-android-9.0.0_r31(9082f2dc) (DO
 NOT SUBMIT)

Android Build Merger (Role) (5):
 - [automerger] Fix CTS regression caused by ag/4690195 am: 73939a628b
 - [automerger] Fix CTS regression caused by ag/4690195 am: 73939a628b
   skipped: 4cfb6100df
 - [automerger] Fix CTS regression caused by ag/4690195 am: 73939a628b
   skipped: 4cfb6100df am: a7b3831011
 - [automerger] Fix CTS regression caused by ag/4690195 am: 73939a628b
   skipped: 4cfb6100df am: a7b3831011 am: 009e9d24ed
 - [automerger] Fix CTS regression caused by ag/4690195 am: 73939a628b
   skipped: 4cfb6100df am: a7b3831011 am: 009e9d24ed am: 902506c4be

Bill Yi (4):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Chad Brubaker (1):
 - Ensure all android.process.media apps have usesCleartextTraffic=true

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

akirilov (4):
 - Fix CTS regression caused by ag/4690195
 - [automerger] Fix CTS regression caused by ag/4690195 am: 73939a628b
   skipped: 4cfb6100df am: a7b3831011 am: 009e9d24ed am: 902506c4be am:
   d04296a513
 - [automerger] Fix CTS regression caused by ag/4690195 am: 73939a628b
   skipped: 4cfb6100df am: a7b3831011 am: 009e9d24ed am: 902506c4be am:
   d04296a513 am: 9606dee784
 - [automerger] Fix CTS regression caused by ag/4690195 am: 73939a628b
   skipped: 4cfb6100df am: a7b3831011 am: 009e9d24ed am: 902506c4be am:
   d04296a513 am: 9606dee784 am: c01d04fa84

android-build-team Robot (5):
 - Snap for 5044688 from 2ab5ac1cdb518f06b86a7777bffb54c81ca119e5 to
   pi-qpr2-release
 - Snap for 5073521 from a66ee95ea755555a9f7fc7e08c1c7a5e355c1c87 to
   pi-qpr2-release
 - Snap for 5087486 from 9e6b883be5731dd7077390aea5b90644832fe3c2 to
   pi-qpr2-release
 - Snap for 5115288 from fa6c3d88ed8cb2ca01a23356bd4a878b55721ed1 to
   pi-qpr2-release
 - Snap for 5130385 from 7316cf114c17cb25fd11bfb3cdbdb8fe303fa42c to
   pi-qpr2-release

Change-Id: I2df3a5c5e13601662914961d3d3f87bcf62f8379

Change-Id: I4795532837d92bcecb8d27671b5b51f01f50bbdb
---
 res/values-bs/strings.xml     | 2 +-
 res/values-iw/strings.xml     | 4 ++--
 res/values-pt-rPT/strings.xml | 2 +-
 ui/res/values-ca/strings.xml  | 2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/res/values-bs/strings.xml b/res/values-bs/strings.xml
index ec099f7b..7b524055 100644
--- a/res/values-bs/strings.xml
+++ b/res/values-bs/strings.xml
@@ -36,7 +36,7 @@
     <string name="download_unknown_title" msgid="7015124071247271585">"&lt;Bez naslova&gt;"</string>
     <string name="notification_download_complete" msgid="5443563299253103667">"Preuzimanje završeno."</string>
     <string name="notification_download_failed" msgid="8612136111952014978">"Preuzimanje neuspješno."</string>
-    <string name="notification_need_wifi_for_size" msgid="2556172885154833575">"Veličina preuzimanja zahtjeva WiFi."</string>
+    <string name="notification_need_wifi_for_size" msgid="2556172885154833575">"Veličina preuzimanja zahtijeva WiFi."</string>
     <string name="notification_paused_in_background" msgid="4328508073283591772">"Pauzirano u pozadini."</string>
     <string name="wifi_required_title" msgid="1995971416871498179">"Preuzimanje je preveliko za mrežu operatera"</string>
     <string name="wifi_required_body" msgid="3067694630143784449">"Morate koristiti WiFi kako biste dovršili ovo preuzimanje od <xliff:g id="SIZE">%s </xliff:g>. \n\nDodirnite <xliff:g id="QUEUE_TEXT">%s </xliff:g> kako biste pokrenuli ovo preuzimanje kada se narednog puta povežete na WiFi mrežu."</string>
diff --git a/res/values-iw/strings.xml b/res/values-iw/strings.xml
index bccb91ad..8ed41165 100644
--- a/res/values-iw/strings.xml
+++ b/res/values-iw/strings.xml
@@ -29,8 +29,8 @@
     <string name="permdesc_seeAllExternal" msgid="1672759909065511233">"‏מאפשר לאפליקציה לראות את כל ההורדות לכרטיס ה-SD, בלי קשר לאפליקציה שהורידה אותן."</string>
     <string name="permlab_downloadCacheNonPurgeable" msgid="3069534308882047412">"שמור מקום במטמון ההורדות"</string>
     <string name="permdesc_downloadCacheNonPurgeable" msgid="2408760720334570420">"מאפשר לאפליקציה להוריד קבצים למטמון ההורדות שלא ניתן למחוק באופן אוטומטי כאשר דרוש למנהל ההורדות שטח נוסף."</string>
-    <string name="permlab_downloadWithoutNotification" msgid="8837971946078327262">"הורד קבצים ללא התראה"</string>
-    <string name="permdesc_downloadWithoutNotification" msgid="8483135034298639727">"מאפשר לאפליקציה להוריד קבצים דרך מנהל ההורדות מבלי להציג הודעות כלשהן בפני המשתמש."</string>
+    <string name="permlab_downloadWithoutNotification" msgid="8837971946078327262">"הורדת קבצים ללא התראה"</string>
+    <string name="permdesc_downloadWithoutNotification" msgid="8483135034298639727">"מאפשר לאפליקציה להוריד קבצים דרך מנהל ההורדות מבלי להציג התראות כלשהן בפני המשתמש."</string>
     <string name="permlab_accessAllDownloads" msgid="2436240495424393717">"גישה לכל הורדות המערכת"</string>
     <string name="permdesc_accessAllDownloads" msgid="1871832254578267128">"מאפשר לאפליקציה להציג ולשנות את כל ההורדות שהופעלו על ידי אפליקציה כלשהי במערכת."</string>
     <string name="download_unknown_title" msgid="7015124071247271585">"‏&lt;ללא כותרת&gt;"</string>
diff --git a/res/values-pt-rPT/strings.xml b/res/values-pt-rPT/strings.xml
index db631cff..4e38cba2 100644
--- a/res/values-pt-rPT/strings.xml
+++ b/res/values-pt-rPT/strings.xml
@@ -28,7 +28,7 @@
     <string name="permlab_seeAllExternal" product="default" msgid="140058400609165726">"Ver todas as transferências para o cartão SD"</string>
     <string name="permdesc_seeAllExternal" msgid="1672759909065511233">"Permite que a aplicação veja todas as transferências efetuadas para o cartão SD, independentemente da aplicação utilizada nas transferências."</string>
     <string name="permlab_downloadCacheNonPurgeable" msgid="3069534308882047412">"Reservar espaço na cache de transferência"</string>
-    <string name="permdesc_downloadCacheNonPurgeable" msgid="2408760720334570420">"Permite que a aplic. transfira ficheiros p/ cache transf., os quais não podem ser automaticamente eliminados quando o gestor transf. necessitar de mais espaço."</string>
+    <string name="permdesc_downloadCacheNonPurgeable" msgid="2408760720334570420">"Permite que a app transfira ficheiros p/ cache transf., os quais não podem ser automaticamente eliminados quando o gestor transf. necessitar de mais espaço."</string>
     <string name="permlab_downloadWithoutNotification" msgid="8837971946078327262">"transferir ficheiros sem notificação"</string>
     <string name="permdesc_downloadWithoutNotification" msgid="8483135034298639727">"Permite que a aplicação transfira ficheiros através do gestor de transferências, sem apresentar qualquer notificação ao utilizador."</string>
     <string name="permlab_accessAllDownloads" msgid="2436240495424393717">"Aceder a todas as transferências do sistema"</string>
diff --git a/ui/res/values-ca/strings.xml b/ui/res/values-ca/strings.xml
index 3892888a..b56b1a86 100644
--- a/ui/res/values-ca/strings.xml
+++ b/ui/res/values-ca/strings.xml
@@ -44,7 +44,7 @@
     <string name="cancel_running_download" msgid="5232704030969221112">"Cancel·la"</string>
     <string name="retry_download" msgid="7617100787922717912">"Torna-ho a provar"</string>
     <string name="start_now_download" msgid="1564642872809509681">"Comença ara"</string>
-    <string name="deselect_all" msgid="6348198946254776764">"Anul·la la selecció de tot"</string>
+    <string name="deselect_all" msgid="6348198946254776764">"Desselecciona-ho tot"</string>
     <string name="select_all" msgid="634074918366265804">"Selecciona-ho tot"</string>
     <string name="selected_count" msgid="2101564570019753277">"S\'han seleccionat <xliff:g id="NUMBER">%1$d</xliff:g> de <xliff:g id="TOTAL">%2$d</xliff:g>"</string>
     <string name="download_share_dialog" msgid="3355867339806448955">"Compart. mitjançant"</string>
-- 
2.17.1

